nvd3/NVD3Chart.py

```diff
@@ -123,6 +124,7 @@ def __init__(self, **kwargs):
         self.height = kwargs.get('height', '')
         self.width = kwargs.get('width', '')
         self.stacked = kwargs.get('stacked', False)
+        self.show_controls = kwargs.get('show_controls', None)
```

Comment:
To not change the current behavior it might be best to set the default for show_controls to `True`, what do you think?